Introduction/README.md

```diff
@@ -385,60 +434,40 @@ by the Docker container `fndemouser/gofn:0.0.2` which may be invoked via the
 specified URL.  Now that we've confirmed deployment was successsful, let's
 call our function.
 
-## Calling Your Deployed Function
-
-There are two ways to invoke your deployed function.  The first is using
-the `fn` CLI which makes invoking your function relatively easy.  Type
-the following:
-
-![User Input Icon](images/userinput.png)
->```sh
-> fn invoke goapp gofn
->```
-
-which results in our familiar output message.
-
-```sh
-{"message":"Hello World"}
-```
-
-Of course this is unchanged from when you ran the function locally.
-However when you called "goapp gofn" the fn server looked up the
-"goapp" application and then looked for the Docker container image
-bound to the "gofn" function.
+### Invoke with Curl
 
-The other way to call your function is via HTTP.  The Fn server exposes our
-deployed function at "http://localhost:8080/t/goapp/gofn-trigger", a URL that
-incorporates our application and function trigger as path elements.
+The other way to invoke your function is via HTTP.  The Fn server exposes our
+deployed function at `http://localhost:8080/t/goapp/gofn-trigger`, a URL
+that incorporates our application and function trigger as path elements.
 
 Use curl to invoke the function:
 
-![User Input Icon](images/userinput.png)
+![user input](images/userinput.png)
 >```sh
 > curl -H "Content-Type: application/json" http://localhost:8080/t/goapp/gofn-trigger
 >```
 
 The result is once again the same.
 
-```sh
+```js
 {"message":"Hello World"}
 ```
 
-We can again pass JSON data to our function and get the value of name passed to the
+We can again pass JSON data to our function get the value of name passed to the
```

Comment:
Not sure why "and" was removed here. "to" maybe? Or stick with "and"?

Reply:
Me either. That's a weird one. Fixed.